core/Updates/5.0.0-rc2.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link    https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Updates;
+
+use Piwik\Common;
+use Piwik\Option;
+use Piwik\Updater;
+use Piwik\Updater\Migration\Factory;
+use Piwik\Updates as PiwikUpdates;
+
+/**
+ * Update for version 5.0.0-rc2
+ */
+class Updates_5_0_0_rc2 extends PiwikUpdates
+{
+    /**
+     * @var Factory
+     */
+    private $migration;
+
+    public function __construct(Factory $factory)
+    {
+        $this->migration = $factory;
+    }
+
+    public function doUpdate(Updater $updater)
+    {
+        $updater->executeMigrations(__FILE__, $this->getMigrations($updater));
+    }
+
+    public function getMigrations(Updater $updater)
+    {
+        $migrations = [];
+
+        // We need to remove all stored view data table settings for referrers evolution chart, as the identifier for
+        // visible rows was changed from label to the type. Keeping the settings would cause no data to be displayed
+        $viewDataTableSettings = Option::getLike('viewDataTableParameters_%_Referrers.getEvolutionGraph');
+
+        foreach ($viewDataTableSettings as $name => $value) {
+            $migrations[] = $this->migration->db->sql(
+                sprintf(
+                    'DELETE FROM %s WHERE option_name = "%s"',
```

Comment:
Just a comment, `option_name` isn't part of an index on the `option` table, can that table have many rows or is it fine to do it this way?

Reply:
`option_name` should be the primary key of that table...